src/System.Net.Sockets/tests/FunctionalTests/AgnosticListenerTest.cs

```diff
@@ -98,6 +98,17 @@ public void ConnectWithV4AndV6_Success()
             listener.Stop();
         }
 
+        [Fact]
+        public void StaticCreate_Success()
+        {
+            TcpListener listener = TcpListener.Create(1234);
+
+            IPEndPoint ep = (IPEndPoint)listener.LocalEndpoint;
+            Assert.Equal(ep.Address, IPAddress.IPv6Any);
+            Assert.Equal(ep.Port, 1234);
+            Assert.True(listener.Server.DualMode);
+        }
```

Comment:
Can we add validation that we can successfully start/stop the listener created this way, or is that unreliable because we need to provide the port explicitly and it could be in use already?